modules/ROOT/pages/_partials/attributes.adoc

```diff
@@ -9,7 +9,7 @@
 
 :service-name:
 
-:mobile-client: Mobile Client
+:mobile-client: Mobile App
```

Comment:
@laurafitzgerald is this a new naming convention we're using?

Reply:
@finp yes, we decided to move from the term client to app with the MDC change as the original blocker to it was that it was a term that was already used in OpneShift